integration/testrunner/db.go

```diff
@@ -42,11 +45,31 @@ func queryAuditLog(ctx context.Context, tb testing.TB, query *bigquery.Query) ([
 	if err != nil {
 		return nil, fmt.Errorf("failed to read job: %w", err)
 	}
-	var row []bigquery.Value
-	if err = it.Next(&row); err != nil {
-		return nil, fmt.Errorf("failed to get next row: %w", err)
+	var results []logpb.LogEntry
+	for {
+		var row []bigquery.Value
+		err := it.Next(&row)
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			return nil, fmt.Errorf("failed to get next row: %w", err)
+		}
+		pbJson := &logpb.LogEntry{}
```

Comment:
Style nit: when you're reflecting into a value, it's usually more common to instantiate a concrete type:

```go
var j logpb.LogEntry
```

and then pass in the pointer:

```go
protojson.Unmarshal([]byte(value), &j)
```